src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -1215,6 +1226,9 @@ $(document).ready(() => {
         deleteQuestion($(e.currentTarget).data('qnnumber'));
     });
 
+    // TODO: Remove this function call after #8488 is fixed
```

Comment:
I think this is supposed to be #8688? I added this in `enableNewQuestion`, I think it's more appropriate there.

Reply:
Oops!! Sorry for the typo. 

>I added this in enableNewQuestion, I think it's more appropriate there.

Moving it to `enableNewQuestion`.